### DISCUSSION TITLE
Arm mode won't turn blue even though every safety check is green

### DISCUSSION THREAD

**Discussion by Pit89** [iNavFlight/inav#6841] [Q&A] [answered]
Hardware
transmitter: Spektrum dx6i
receiver: spm4650
FC: mamba f405 mk2 stack 
firmware: furyf40sd
gps compass: matek M8Q-5883
barometer: bmp180

Hey, a couple of weeks ago i decided to get into the fpv hobby.  I bought all the pieces and successfully soldered them to the fc. Unfortunately i'm having some problems, for the last couple of days i tried to look for the answer on google but i haven't found it.
The problem i'm running into is in the modes, when i try to flip the switches on my transmitter, the boxes don't turn blue. It happens to pos_hold, rth (even with 3d fix) and arm but weirdly the buzzer works. I don't know if the problem is related to the transmitter, some settings that i overlooked or hardware issues.
I also noticed that if i set the modes in order to have the stick inside the range and try to arm the drone(without having it connected to my laptop via usb) the blue light on my fight controller goes from blinking slowly( 1 blink a second) to really fast.
I'm attaching also the status and dump readouts.
Hopefully someone knows what's going on.
 Thanks in advance, Pietro.
[Logs.txt](https://github.com/iNavFlight/inav/files/6325619/Logs.txt)

**Comment by avsaase** [comment, marked as answer]
You cannot arm into a navigation mode. You also cannot enable a navigation mode when you are disarmed, so the box in the configurator won't light up.

**Reply by Pit89** [reply]
Thanks for the reply, but if I just arm the quad, it shouldn't go into any navigation mode... And it still doesn't make the motors spin in my case.

**Reply by avsaase** [reply]
> Arming disabled flags: NAV RX CLI

This says you cannot arm because you have a navigation mode selected and you have no RC link (and that you have the CLI enabled but that's not the problem here).

**Comment by Pit89** [comment]
Can anyone help me?

**Comment by widar123** [comment]
For some time now, I've been struggling with a problem I encountered in the INAV configurator (8.0.1) working with a Speedybee F405 wing controller (firmware 8.0.1). The GPS is configured correctly, works, and finds a 3D fix with four satellites, but its status is always blue regardless of the number of satellites. This is a bit annoying because it's unclear whether the GPS has a fix or not. The only place I can check this is the GPS tab, where I can see the current satellite count and a map with the aircraft's exact location. From what I understand, this status should change from red (when there is no fix), yellow (when the number of satellites is too low), and blue or green (when the GPS has a fix with enough satellites for a 3D fix). What could be the cause? Unfortunately, searching online hasn't found any solutions.
